styles.css

```diff
@@ -883,3 +895,235 @@ input:focus {
     border-color: var(--color-secondary);
   }
 }
+
+/* Popup */
+.tag-items {
+  box-sizing: border-box;
+  padding: 1rem 1.2rem;
+  background: #fff;
+  border: 1px solid #8993a4;
+  font-family: var(--font-inter);
+  font-style: normal;
+  font-weight: 500;
+  font-size: 1.5rem;
+  line-height: 2rem;
+  color: #3c3a39;
+}
+
+/* Popup */
+
+.popup-modal {
+  z-index: 999;
+  position: fixed;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+  width: 100%;
+  background-color: rgba(0, 0, 0, 0.5);
+  backdrop-filter: blur(0.1rem);
+  transition: all 0.5s ease;
+  top: 0;
+}
+
```

Comment:
- [ ] It is required to produce a close replica of the figma design. Kindly check how your design displays on the mobile with how it should be on the figma. Some content of your modal on mobile are not properly visible. Kindly adjust your styling to make them visible as we have them on the Figma design.

### Your Design

![your-mobile-popup](https://github.com/peteryeungtt/My-Portfolio/assets/26597432/2f789792-8b57-4864-bd63-998bd6c2d2ce)

### Figma Design
![temp-mobilepopup](https://github.com/peteryeungtt/My-Portfolio/assets/26597432/523a1fe1-4d50-47ba-906e-190566ee939f)